python/cuml/solvers/cd.pyx

```diff
@@ -296,7 +296,7 @@ class CD(Base,
                       <double>self.tol,
                       <double*>sample_weight_ptr)
 
-            self.intercept_ = _c_intercept2_f64
+                self.intercept_ = _c_intercept2_f64
```

Comment:
Unrelated fix?

Reply:
yeah, it just seemed way to small of a change to warrant a full PR with CI run